packages/@sanity/form-builder/src/inputs/PortableText/PortableTextInput.tsx

```diff
@@ -84,12 +120,16 @@ const PortableTextInputWithRef = React.forwardRef(function PortableTextInput(
 
   const toast = useToast()
 
-  const editorId = useMemo(() => uniqueId('PortableTextInputRoot'), [])
   // Memoized patch stream
   const patches$: Subject<EditorPatch> = useMemo(() => new Subject(), [])
   const patchObservable = useMemo(() => patches$.asObservable(), [patches$])
 
-  const handleToggleFullscreen = useCallback(() => setIsFullscreen(!isFullscreen), [isFullscreen])
+  const innerElementRef = useRef(null)
```

Comment:
Should be typed so it’s easier to migrate to strict TypeScript in the future:

```suggestion
  const innerElementRef = useRef<HTMLDivElement | null>(null)
```